kymatio/scattering3d/tests/test_scattering3d.py

```diff
@@ -274,42 +274,6 @@ def test_scattering_methods():
     scattering.rotation_covariant = False
     Sx = scattering(x)
 
-def test_cpu_cuda():
```

Comment:
Can't you modify this test to work? All you have to do is remove the `assert S.is_cuda`

Reply:
This test is redundant(since I removed the local variable is_cuda) with most of the other subparts of the code, I don't want to keep it!